source/ranges.tex

```diff
@@ -866,6 +866,12 @@
 \end{itemdecl}
 
 \begin{itemdescr}
+\pnum
+\tcode{T} models \libconcept{SizedRange} only if its
+copy constructor, move constructor (if any),
+copy assignment operator, and move assignment operator (if any)
+are \bigoh{1}.
+
```

Comment:
What does O(1) refer to? The number of bytes taken up by the range object?
Where is the existing normative statement that says this? If there is none, this change does not seem editorial.

Reply:
The [range.view] p1 sentence is much clearer than the O(1) we have here; it clearly says "not proportional to the number of elements in the View". Can we find a formulation that actually says this? Maybe "... only if the execution time required for the initializations and assignments required by Copyable<T> is independent of the number of elements in the View"? This has one "required" too many, and maybe the library says something else for "execution time" when it wants to talk about time complexity.

Reply:
Right. I used that "that is" explanation as part of your suggested fix.